.github/workflows/ci.yaml

```diff
@@ -6,14 +6,33 @@ on:
       - 'main'
   pull_request:
 
+env:
+  CARGO_INCREMENTAL: 0
+  RUSTFLAGS: "-Dwarnings"
+  CARGO_UNSTABLE_SPARSE_REGISTRY: true
+
 jobs:
   buf-lint:
     runs-on: ubuntu-22.04
     steps:
       - uses: actions/checkout@v3
       - uses: bufbuild/buf-setup-action@v1.13.1
+        with:
```

Comment:
Was seeing rate limiting failures [quite a bit for `buf-lint`](https://github.com/mobilecoinfoundation/protobufs/actions/runs/4165683357/attempts/1). Based on https://github.com/bufbuild/buf-setup-action#input Hoping this will address that[